README.md

```diff
@@ -725,3 +725,55 @@ Error Connecting to auth service ...
 }
 ```
 
+
+
+
+### Get Importer Mappings
+
+This endpoint returns a list of available staging importer apps that have been whitelisted. This endpoint also
+returns a mapping between a list of files and predicted importer app.
```

Comment:
Wouldn't it be better to split this out into different endpoints? I'm not fond of endpoints that send back a mishmash of different things, especially when you're preventing the endpoint from returning potentially useful information about erroneous input (e.g. input in the wrong format). If I were designing this service, I would have a GET endpoint that returns the list of allowed importer apps, and a POST endpoint that you submit a file list to, and it returns a list of mappings between valid file names and the predicted importer app. The GET endpoint always returns 200 (unless the service is offline), but the POST endpoint will return a 400 if the input is in the wrong format or there's something wrong with any of the files in the input list.

Reply:
@ialarmedalien 
I don't understand your comment about the wrong input. 
Can you give an example of erroneous input? I couldn't think of any?

Reply:
Oh, I mean I guess if we did something like 

    data = {"apple": [{}]}
    async with AppClient(config, username) as cli:
        resp = await cli.post(
            "importer_mappings/", headers={"Authorization": ""}, data=data
        )

It will crash with "IndexError: list index out of range"

Reply:
I can make it instead crash like so

```
    if len(file_list) == 0:
        raise web.HTTPBadRequest(text="must provide file_list field ")
```

Reply:
re: "This endpoint returns a list of available staging importer apps that have been whitelisted."

Can you alter the text description to (a) not use the word "whitelist" (racist overtones), and (b) explain what the apps are whitelisted for? I think just saying "This endpoint returns a list of apps for importing files to the staging service" would probably work, although you can add a bit more explanation if you like.